cloudlift/deployment/task_definition_creator.py

```diff
@@ -100,9 +100,13 @@ def update(self):
         log_bold("Task definition successfully updated\n")
 
     def _current_task_defn(self, ecs_client: EcsClient, deployment: DeployAction):
+        if not ecs_client.list_task_definitions(self._task_defn_family()):
+            log_err("Task definition was created with the help of the create_service\ntry updating with 'update_service' command")
```

Comment:
This task definition was created for a service. Please use `cloudlift update_service` to modify it.